### PR TITLE
fix: remove showcases section from jumper

### PR DESCRIPTION
**Pull request by Abhikumar98**
CleanShot 2023-09-08 at 16 11 34@2x

### CODE REVIEW

File: packages/dapp/src/const/MenuConfig/useDevelopersContent.tsx
Comment (dennyscode) [inline packages/dapp/src/const/MenuConfig/useDevelopersContent.tsx:1]: Two more imports can be deleted from here:

```
import SlideshowIcon from '@mui/icons-material/Slideshow';
import { MenuKeys } from '..';
```

My linter automatically removed it, how come it´s not the case on your machine? Let´s check if it´s properly set up as I had noticed a few times that some Linter-suggestions popped up which might not have been shown to you.

**Review by dennyscode — CHANGES_REQUESTED** [review]
Let´s also delete the showcases labels from the english-translation file. 


```
   "developers": {
      "github": "GitHub",
      "documentation": "Documentation",
      ~~"showcases": "Showcases"~~
    },

~~    "showcases": {
      "ukraineDonation": "Support Ukraine 🇺🇦",
      "klimaStaking": "Cross Chain Klima Staking",
      "carbonOffset": "Cross Chain Carbon Offset"
    }~~
```

**Comment by Abhikumar98** [comment]
> Let´s also delete the showcases labels from the english-translation file.
> 
> ```
>    "developers": {
>       "showcases": "Showcases"
>     },
> 
>     "showcases": {
>       "ukraineDonation": "Support Ukraine 🇺🇦",
>       "klimaStaking": "Cross Chain Klima Staking",
>       "carbonOffset": "Cross Chain Carbon Offset"
>     }
> ```

I was kinda not sure about removing it manually or does it sync with Crownin, hence didn't touch it. Should I go ahead with it?

File: packages/dapp/src/const/MenuConfig/useDevelopersContent.tsx
Comment (Abhikumar98) [reply packages/dapp/src/const/MenuConfig/useDevelopersContent.tsx:1]: ahh, got it.
It misses it sometimes when there's a lag. Removing it manually

**Comment by dennyscode** [comment]
From my understanding: The english-translation file acts as a baseplate. Crowdin picks up on the changes within that file from our repo and represents that list on the Crowdin-platform. So, deleting it here should do no harm and just cleans up our "dictionary" - in code and on our active translations on crowdin.

It´s probably still being cached/saved on our Crowdin account, so whenever we need to use it again, Crowdin will likely recommend us our past translations.

@chybisov Please confirm

**Comment by chybisov** [comment]
@dennyscode 😄
ggPUBI0

**Review by dennyscode — APPROVED** [review]

**Review by chybisov — APPROVED** [review]

**Pull request merged**